Misc/NEWS.d/next/Library/2021-12-14-21-19-04.bpo-46075.KDtcU-.rst

```diff
@@ -0,0 +1 @@
+``CookieJar`` with ``DefaultCookiePolicy`` now can process cookies from localhost with domain=localhost explicitly specified in Set-Cookie header.
```

Comment:
CI is failing because there's no trailing newline. I tried to edit it from the web interface but wasn't successful, could you add it?

Reply:
It looks like you added it? The CI passes.

Reply:
Ah right, I refreshed the page and no commit from me showed up, but maybe GH was just lagging.